lib/marked.js

```diff
@@ -39,10 +39,6 @@ block.list = replace(block.list)
   ('def', '\\n+(?=' + block.def.source + ')')
   ();
 
-block.blockquote = replace(block.blockquote)
```

Comment:
this is plain wrong, cannot remove this without changing the block.blockquote regex